### ISSUE TITLE
[Bug]: FTB Quests - Quests triggers multiple-times upon completion

### ISSUE THREAD

**Issue by TheonlyTazz** [FTBTeam/FTB-Mods-Issues#955]
### Mod

FTB Quests

### Mod version

ftb-quests-forge-1802.3.15-build.238.jar

### Forge / Fabric version

1.18.2 / 40.2.0

### Modpack & version

- 

## Issue
Quests that are repeatable, with auto-claimed rewards and submittable items are finished multiple times, as long as you have items inside your inventory

---
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/3908847/80eb3e7d-bd2f-432b-9980-173219bf74b7)
---
### Task
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/3908847/3911b7aa-7a96-4305-a419-7d0c6d29d003)
---
### Reward
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/3908847/e595ca06-e2aa-4e15-984f-5614d65b0d68)
---
### Quest
![image](https://github.com/FTBTeam/FTB-Mods-Issues/assets/3908847/5a35d6a3-58a2-4aef-a77a-0a215bcf6652)

## To replicate: 
 -  Setup a Quest, which needs items to submit, set the reward to Auto-claim and the quest on repeatable
  - have a multiple of the items needed.
  -  Submit once => all Items get consumed and the quest is completed multiple times, resulting in all the multiples of the submited items to be removed and multiple of the rewards being given

**Comment by desht** [comment]
First question: did you set the task to Consume Items = true? That would be fairly important; without that the quest will continually auto-submit and grant rewards. With Consume Items on, the player also has to open the task and press "Submit Items", which should prevent this repeated completion.  At least that's what happened when I set up a quest like you described.

**Comment by TheonlyTazz** [comment]
Yes, the quests are setup exactly as shown above, and upon clicking the submit button on the task popup, it will consume all items
